gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```

Comment:
Does this need to be in the repo?

Reply:
My understanding of the best practice is to include gradle with the wrapper in the repo.  Besides eliminating the need for the user to install gradle, it also means your gradle version and build script are in sync.  So, if a user has multiple projects with gradle build scripts from different versions, they don't need to install multiple versions of gradle and choose between them when they build the project.